privacyidea/static/components/dialogs/views/dialog.lock.html

```diff
@@ -20,7 +20,6 @@ <h4 class="modal-title" id="myModalLock" translate>
                         <label for="password" class="sr-only"
                                translate>Password</label>
                         <input name="password" type="password"
-                               autocomplete="off"
```

Comment:
For this component I am not quite sure if the browser should pre-fill this, I guess it is a login from to the server?

Reply:
This is the password dialog which blocks when the logout time has been reached. I think it is ok to enable autocomplete here.